projects/objects/traffic/protos/CautionPanel.proto

```diff
@@ -18,8 +18,8 @@ PROTO CautionPanel [
     recognitionColors IS recognitionColors
     children [
       Transform {
-        translation 0.008 -1.73 0
-        rotation 0 0 1 0
+        translation 0 -0.008 -1.73
+        rotation -0.577352 0.577348 0.577352 -2.094405
```

Comment:
On line 82 it can't be done as the Box is supposed to be sideways, but there might be cases like that yes.
Often times I did indeed change the transform instead of the coordinates, the reason being that if the Transform still needs to be there (for a translation for instance), whether we rotate by the default value or by a non-default one, the performance will be the same so it seemed more clear. But I'll double check to see if there are cases where the Transform could be removed, in which case it might be better to collapse it.